docs/config.yaml

```
@@ -113,17 +115,38 @@ s3:
 # OPTIONAL: container registry-related keys
 registry_repos:
   # OPTIONAL: repo to which to push oscontainer
```

Minor: maybe this now can be:

```suggestion
  # OPTIONAL: repo and tags to which to push oscontainer
```